akka-cluster-tools/src/main/scala/akka/cluster/pubsub/DistributedPubSubMediator.scala

```diff
@@ -251,6 +238,36 @@ object DistributedPubSubMediator {
     @SerialVersionUID(1L)
     final case class SendToOneSubscriber(msg: Any)
 
+    /*
+     * Messages used to encode protocol to make sure that we do not send Subscribe/Unsubscribe message to 
+     * child (mediator -> topic, topic -> group) during a period of transition. Protects from situations like:
+     * 
+     * Sending Subscribe/Unsubscribe message to child actor after child has been terminated 
+     * but Terminate message did not yet arrive to parent.
+     *
+     * Sending Subscribe/Unsubscribe message to child actor that has Prune message queued and pruneDeadline set.
+     *
+     * In both of those situation parent actor still thinks that child actor is alive and forwards messages to it resulting in lost ACKs.
+     */
+    /**
+     * Passivate-like message sent from child to parent, used to signal that sender has no subscribers and no child actors.
+     */
+    case object NoMoreSubscribers
+
+    /**
+     * Sent from parent to child actor to signalize that messages are being buffered. When received by child actor
+     * if no [[Subscribe]] message has been received after sending [[NoMoreSubscribers]] message child actor will stop itself.
```

Comment:
the javadoc problems might also be because of these links, (objects in objects). I suggest that you just change to `Subscribe` with single back-tick, same with other such links here and below